### PR TITLE
test: Add flank multi test targets example

### PR DESCRIPTION
**Pull request by jan-gogo**
This PR adds FlankMultiTestTargetsExample required in #1281. 
Is created for reducing code review complexity of #1281.

### CODE REVIEW

**Review by pawelpasterz — APPROVED** [review]

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
```
Comment (Sloox) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift:6]: spacing?

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        // Use this method to select a configuration to create the new scene with.
+        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
+    }
+
+    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+        // Called when the user discards a scene session.
+        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+    }
+
```
Comment (Sloox) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift:26]: same here? extra spaces?

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/SceneDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+
```
Comment (Sloox) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/SceneDelegate.swift:7]: ;)

**Review by Sloox — CHANGES_REQUESTED** [review]
remove some spaces?

**Review by piotradamczyk5 — APPROVED** [review]

**Comment by piotradamczyk5** [comment]
I changed title to **test:** prefix

File: test_projects/ios/FlankMultiTestTargetsExample/UITests/UITests.swift [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    }
+    
+    func test1ENLocale() {
+        XCTAssert(Locale.current.identifier.contains("en"))
+    }
+    
+    func test1PLLocale() {
+        XCTAssert(Locale.current.identifier.contains("pl"))
+    }
+}
```
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/UITests/UITests.swift:24]:
```suggestion
}

```

File: test_projects/ios/FlankMultiTestTargetsExample/SecondUITests/SecondUITests.swift [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    }
+    
+    func test2ENLocale() {
+        XCTAssert(Locale.current.identifier.contains("en"))
+    }
+    
+    func test2PLLocale() {
+        XCTAssert(Locale.current.identifier.contains("pl"))
+    }
+}
```
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/SecondUITests/SecondUITests.swift:24]:
```suggestion
}

```

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/SceneDelegate.swift [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        // Use this method to undo the changes made on entering the background.
+    }
+
+    func sceneDidEnterBackground(_ scene: UIScene) {
+        // Called as the scene transitions from the foreground to the background.
+        // Use this method to save data, release shared resources, and store enough scene-specific state information
+        // to restore the scene back to its current state.
+    }
+
+
```
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/SceneDelegate.swift:43]:
```suggestion
```

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/SceneDelegate.swift (hunk 3 above)
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/SceneDelegate.swift:7]:
```suggestion
```

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift:27]:
```suggestion
```

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift [hunk 8]
```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
```
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/AppDelegate.swift:7]:
```suggestion
```

File: test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/ViewController.swift [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+
```
Comment (jan-gogo) [inline test_projects/ios/FlankMultiTestTargetsExample/FlankMultiTestTargetsExample/ViewController.swift:10]:
```suggestion
```

**Review by piotradamczyk5 — APPROVED** [review]

**Review by Sloox — APPROVED** [review]

**Pull request merged**